### DISCUSSION TITLE
More Details on Tire Pressure Warning

### DISCUSSION THREAD

**Discussion by MrPaulAR** [teslamate-org/teslamate#3611] [Ideas]
Is it possible to have this tooltip to include the pressure from the warning even if the notification from Tesla on includes this level of detail?  

![image](https://github.com/teslamate-org/teslamate/assets/3765696/5453f5bd-b490-4545-abf1-a2894082bc8a)

**Comment by NirKli** [comment]
PR created to add this: https://github.com/teslamate-org/teslamate/pull/4654
Shows tire names, pressure values, and supports PSI/Bar. Let me know what you think!

**Comment by JakobLichterfeld** [comment]
solved by #4654 which will be included in next release
